pkg/pb/metainfo.proto

```diff
@@ -401,12 +401,12 @@ message SegmentBeginResponse {
 message SegmentCommitRequest {
     bytes segment_id = 1 [(gogoproto.customtype) = "SegmentID", (gogoproto.nullable) = false];
 
-    bytes encrypted_key_nonce = 2 [(gogoproto.customtype) = "Nonce", (gogoproto.nullable) = false];
-    bytes encrypted_key = 3;
+    bytes encrypted_key_nonce = 3 [(gogoproto.customtype) = "Nonce", (gogoproto.nullable) = false];
```

Comment:
why the number change here?

Reply:
oh, I moved `position` back to `SegmentBeginRequest` and I forgot to adjust numbers